robot/Cumulus/tests/browser/batch_gift_entry/bge_default_batch.robot

```diff
@@ -1,6 +1,7 @@
 *** Settings ***
 
-Resource        tests/NPSP.robot
+#Resource        robot/Cumulus/resources/NPSP.robot
```

Comment:
looks like you forgot to remove the commented line

Reply:
Thanks for catching that. I've removed it and pushed the change.

Reply:
Thanks for catching that. I've removed it and pushed it.